src/features/Instructor/Profile/index.jsx

```diff
@@ -83,6 +85,9 @@ const Profile = () => {
             )}
           </ProfileCard>
         </Tab>
+        <Tab eventKey="availability" title="Availiability">
```

Comment:
On hover I loose the bottom line
![image](https://github.com/user-attachments/assets/2ac0b870-d1ba-4ce9-b8a7-2b0ff684cc17)